test/behavior/byteswap.zig

```diff
@@ -3,7 +3,36 @@ const builtin = @import("builtin");
 const expect = std.testing.expect;
 
 test "@byteSwap integers" {
-    if (builtin.zig_backend == .stage2_wasm) return error.SkipZigTest;
+    if (builtin.zig_backend == .stage2_wasm) {
```

Comment:
In the future, it'd be easier to just add a new test function that skips for all the backends but the Wasm backend which we can just remove once it's no longer needed. It's ok to leave this for now.